dev/ci/scripts/utils/ci_utils.sh

```diff
@@ -177,6 +177,44 @@ function publish_logs() {
     echo "${URL}"
 }
 
+function publish_logs_from_file() {
+  # publish_logs_from_file
+  # Reads a file that lists relative file names (one per line) under a directory
+  # and publishes them using publish_logs.py. When more than one valid file is
+  # found, call the python utility with --multiple --format github for gist
+  # output formatting.
+  # Usage: publish_logs_from_file <ID> <list_file>
+  local PR_header="$1"
+  local list_file="$2"
+  local files=()
+
+  # Read the list file and build an array of existing full paths
+  while IFS= read -r line || [[ -n "${line}" ]]; do
+    # skip empty lines
+    [[ -z "${line// /}" ]] && continue
+    if [[ -f "${line}" ]]; then
+      files+=("${line}")
+    else
+      echo "File ${line} does not exist"
+    fi
+  done < "${list_file}"
+
+  local URL=""
+  if (( ${#files[@]} )); then
+    # First, upload to repo (retain original behavior) if desired
+    # shellcheck disable=SC2027,SC2086
+    ${HOMEgfs_}/dev/ci/scripts/utils/publish_logs.py --file "${files[@]}" --repo ${PR_header} > /dev/null || true
```

Comment:
I don't think you want to disable either 2027 or 2086 here.  `PR_header` and `HOMEgfs_` are simple strings:

```suggestion
    "${HOMEgfs_}/dev/ci/scripts/utils/publish_logs.py" --file "${files[@]}" --repo "${PR_header}" > /dev/null || true
```

Reply:
Oh I used to have to not use the quotes on the files argument.  Let me try to remove the SC's them again and add the quotes on the first part.